sql/planbuilder/set.go

```diff
@@ -200,6 +200,10 @@ func (b *Builder) buildSysVar(colName *ast.ColName, scopeHint ast.SetScope) (sql
 			}
 			return sysVar, scope, true
 		default:
+			// TODO: currently using table field to identify postgres configuration parameters, but need fix
+			if table == "postgres" {
```

Comment:
We should try and find a better way of doing this before merging it in. This could break a GMS/Dolt user who has a table named `"postgres"`. I don't know _why_ they would, and there might not be anyone, but there is the potential and we can't concretely say that nobody is.